creativecloud/scripts/utils.js

```diff
@@ -62,6 +62,19 @@ function defineDeviceByScreenSize() {
   return 'TABLET';
 }
 
+function heroForegroundImage(firstBlock) {
+  let rows = firstBlock.querySelectorAll(':scope > div');
+  if (rows.length > 1 && rows[0].textContent !== '') {
+    const [, ...tail] = rows;
+    rows = tail;
+  }
```

Comment:
we can remove const [, ...tail] = rows;
    rows = tail;
  }

Reply:
I don't this is needed as shift() will remove the head row.